queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animal WHERE name LIKE '%mon';
+
+SELECT name from animal WHERE date_of_birth  >= '2016-01-01' AND date_of_birth <= '2019-01-01';
```

Comment:
- [ ] I think If you want to select all animals born between 2016 and 2019, the end date of the range should be December 31, 2019. The correct SQL query would be: `SELECT name FROM animal WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-12-31';
`

Reply:
@PinkMoon25 Thank you. Will make the required changes